app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%=render "shared/navbar", right_el: "menu", left_el: "none", title: "TRANSACTIONS"%>
+<div class="bg-secondary text-light"><span>Total Amount: $100, Total Items: 9</span></div>
+    <ul class="category-list d-flex flex-column align-items-center m-auto  p-3">
+        <%for a in 0..30 do%>
+            <%= render "transactions/shared/transaction"%>
+        <%end%>
+    </ul>
```

Comment:
- [ ] Great job fixing the buttons in the app as requested by the previous reviewer but there's a issue in the transaction page, If I add some transactions and they exceed the height of the page I'm not able to see the total amount it's underflowing the `add new transaction` button. Please make sure to fix this issue (You can reference the categories page that is working fine in the same situation.

![image](https://github.com/nilab-khwaja/budget-app/assets/83803414/29128c13-a907-41a2-b0a0-86f0b1ab7ca6)